packages/commons-ui-core/styles/createTheme/createTheme.js

```diff
@@ -0,0 +1,25 @@
+import { createTheme as MuiCreateTheme } from "@mui/material/styles";
+import { deepmerge } from "@mui/utils";
+
+const createTheme = (options = {}) => {
+  const { breakpoints, contentWidths } = options;
+
+  if (breakpoints && contentWidths) {
+    return MuiCreateTheme(options);
+  }
+  return MuiCreateTheme(
+    deepmerge(options, {
+      contentWidths: {
+        values: {
+          xs: 0,
+          sm: 480,
+          md: 608,
+          lg: 1160,
+          xl: 1440,
+        },
+      },
+    })
```

Comment:
incase `contentWidths` is defined in options, then we should have `{ contentWidths: {} }` followed by `options` in the deepmerge.

Reply:
Not sure I understood this - the `contentWidths` option is the default given if one is not passed...

Reply:
@gertie-sheshe I see the code above it (`if statement`) has changed, so it is now alright. Initially, the if statement above was checking for both `breakpoints && contentWidths` ...meaning if just contentWidths was defined on options without breakpoints it would be overridden by the default. That's why I had suggested the default contentWidths appear as the first parameter of `deepmerge`